packages/invoices/src/InboundShipment/DetailView/columns.ts

```diff
@@ -148,3 +148,13 @@ export const useInboundShipmentColumns = (): Column<
     {},
     []
   );
+
+export const useExpansionColumns = (): Column<InvoiceLine>[] =>
+  useColumns([
+    'batch',
+    'expiryDate',
+    'locationName',
+    'numberOfPacks',
+    'packSize',
+    'costPricePerPack',
+  ]);
```

Comment:
wasn't sure about the columns 🤔

Reply:
given how wrong I was last time, best to jam something in and ask for a review - so this is all good